src/app.ts

```diff
@@ -36,6 +36,12 @@ const cli = new Cli({
         reg.setAppServiceToken(AppServiceRegistration.generateToken());
         reg.setSenderLocalpart("slackbot");
         reg.addRegexPattern("users", `@${config.username_prefix}.*:${config.homeserver.server_name}`, true);
+        const teamSyncEntries = config.team_sync && Object.values(config.team_sync) || [];
+        for (const teamEntry of teamSyncEntries) {
+            if (teamEntry.channels?.alias_prefix) {
+                reg.addRegexPattern("aliases", `#${teamEntry.channels?.alias_prefix}.*:${config.homeserver.server_name}`, true);
+            }
+        }
```

Comment:
This allows the app service to control all aliases matching that regex?. Good to know.

You mention this being a breaking change. That's because as long as admins don't re-generate the registration file, the appservice doesn't have permission to create/modify any aliases, right?